keyboards/idb/idb_60/idb_60.h

```diff
@@ -3,18 +3,18 @@
 #include "quantum.h"
 
 inline void _idb_60_caps_led_on(void) {
-    writePinLow(C5);
+    writePinHigh(C5);
 }
 
-inline void _idb_60_esc_led_on(void) {
-    writePinLow(C4);
+inline void _idb_60_fn_led_on(void) {
+    writePinHigh(C4);
 }
 
 inline void _idb_60_caps_led_off(void) {
     writePinLow(C5);
 }
 
-inline void _idb_60_esc_led_off(void) {
+inline void _idb_60_fn_led_off(void) {
     writePinLow(C4);
 }
 
```

Comment:
Maybe some backwards compatibility here, depends how many boards are in the wild with keymaps out of the repo?
```suggestion

#define _idb_60_esc_led_on _idb_60_fn_led_on
#define _idb_60_esc_led_off _idb_60_fn_led_off

```